packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -248,43 +257,35 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     }
   }
 
-  private _onFocus = (ev: React.FocusEvent<HTMLInputElement>): void => {
-    // We can't set focus on a non-existing element
+  private _handleFocus = (ev: React.FocusEvent<HTMLInputElement>): void => {
```

Comment:
I believe most event handlers use `_on` prefix in the codebase; why introduce inconsistency?